docs/HowTo/Get-Started/Starting-node.md

```diff
@@ -94,6 +94,20 @@ rpc-http-enabled=true
 data-path="/tmp/tmpdata-path"
 ```
 
+!!! caution
+
+    We do not recommend using the following settings in a production environment:
```

Comment:
Instead of "we" having the sentence like "There's a security risk to use the following settings in a production environment"

Reply:
Using "we" is actually a standard we've taken from the Microsoft MoS (our default standard) - https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/recommend 

Having said that, now we're open source I'm not sure that still makes sense.  Who is 'we'? 

@NicolasMassart

Reply:
Yes, quite a few instances of "We" in my travels through the Besu docs. I must admit I've never been a fan of using "we" for the reason you state, @MadelineMurray. Who is "we"?
Perhaps, "The following settings pose a security risk if used in production environments:"?

If I can raise another question about these statements? I've been using "production environment", but I've also seen "production code", and other variations on the same thing. Is there a phrase we want to standardise on?

Reply:
On the second question: production environment is the term. There's no production code, the code, once released, is the same whatever you use it for. There's a non released code too, but our should talk only about the released one IMO.

Reply:
Changed to "The following settings are a security risk in production environments:"